login-page.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+</head>
+<body>
+    <fieldset>
+        <legend>Login Authentication</legend>
+        <form action="">
+            <br>
+            <label for="mail">Email:</label>
```

Comment:
Raj please explain the procedures using comment line

Reply:
Resolved